src/components/sidebar/sidebar.component.js

```diff
@@ -25,13 +27,31 @@ class Sidebar extends Modal {
             data-element='close'
             onClick={ this.props.onCancel }
             type='close'
+            tabIndex='0'
+            onKeyDown={ ev => (Events.isEnterKey(ev) ? this.props.onCancel() : null) }
```

Comment:
This should have role=button and should listen for Enter and Space. All close icons should be the same.